### ISSUE TITLE
Ord SEND error with Insufficient Funds

### ISSUE THREAD

**Issue by dkofoed** [ordinals/ord#3695]
I can't figure out what i'm doing wrong and would appreciate any input.  I'm trying to SEND the remaining cardinal amount from my ORD wallet to a different wallet address.  No matter what amount of sats I select (or fee rate) I always get an "Insufficient funds" error.

This is the amount in the ord wallet:

{
  "cardinal": 2700200,
  "ordinal": 0,
  "runes": {},
  "runic": 0,
  "total": 2700200
}

I'm trying with this command:

ord --cookie-file "D:\bitcoin-26.1\Bitcoin\.cookie" wallet send --fee-rate 40 3BDRJ8EMsdCZJ6PFJQCSdhBkpx4xn8Ho9H 2000000sats

What am I missing?

**Comment by cryptoni9n** [comment]
Hi dkofoed,

This issue could be due to a few problems causing the wallet.  If you run `ord wallet transactions` do you see any that are outstanding and have 0 confirmations?  If you run `bitcoin-cli -datadir=<DATA_DIR> -rpcwallet=<WALLET> listunspent` and total up each, does it equal 2700200?

**Comment by dkofoed** [comment]
Does this look like the correct command?

bitcoin-cli -datadir="D:\bitcoin-26.1\Bitcoin" -rpcwallet bc1pvxn8evmrfnvctwhypsw4qc95he02ckyqrnvu3y7zpg34xy2k6t4sw50d4t listunspent

I get:
error code: -32601
error message:
Method not found

**Comment by cryptoni9n** [comment]
> Does this look like the correct command?
> 
> bitcoin-cli -datadir="D:\bitcoin-26.1\Bitcoin" -rpcwallet bc1pvxn8evmrfnvctwhypsw4qc95he02ckyqrnvu3y7zpg34xy2k6t4sw50d4t listunspent
> 
> I get: error code: -32601 error message: Method not found

no, you must specify the name of your ord wallet, not the address, in the rpcwallet flag, which is probably ord.  so try something like `bitcoin-cli -datadir="D:\bitcoin-26.1\Bitcoin" -rpcwallet=ord listunspent`

**Comment by dkofoed** [comment]
cool, got that to work and after adding up the UXTOs they add up to 0.027002 which seems correct.  I ran the transactions command as well and get this:

  {
    "transaction": "f9a7520f1a2e6c14b1e69b20080d85aa7a03e6c82b5a7c9baf509f3ed2425feb",
    "confirmations": 1047
  },
  {
    "transaction": "a2bc96901493f0116ae7f375d1ed023a8b6d1db3f00e0a74cc4bbd1dc88a576b",
    "confirmations": 1031
  },
  {
    "transaction": "a2bc96901493f0116ae7f375d1ed023a8b6d1db3f00e0a74cc4bbd1dc88a576b",
    "confirmations": 1031
  },
  {
    "transaction": "a2bc96901493f0116ae7f375d1ed023a8b6d1db3f00e0a74cc4bbd1dc88a576b",
    "confirmations": 1031
  },
  {
    "transaction": "a2bc96901493f0116ae7f375d1ed023a8b6d1db3f00e0a74cc4bbd1dc88a576b",
    "confirmations": 1031
  },
  {
    "transaction": "a2bc96901493f0116ae7f375d1ed023a8b6d1db3f00e0a74cc4bbd1dc88a576b",
    "confirmations": 1031
  },
  {
    "transaction": "a2bc96901493f0116ae7f375d1ed023a8b6d1db3f00e0a74cc4bbd1dc88a576b",
    "confirmations": 1031
  },
  {
    "transaction": "a2bc96901493f0116ae7f375d1ed023a8b6d1db3f00e0a74cc4bbd1dc88a576b",
    "confirmations": 1031
  },
  {
    "transaction": "a2bc96901493f0116ae7f375d1ed023a8b6d1db3f00e0a74cc4bbd1dc88a576b",
    "confirmations": 1031
  },
  {
    "transaction": "a2bc96901493f0116ae7f375d1ed023a8b6d1db3f00e0a74cc4bbd1dc88a576b",
    "confirmations": 1031
  },
  {
    "transaction": "fdea7686f3437eaffdb927ed87f06102455b4049e8e985f56b8bd6a5693aabd5",
    "confirmations": 410
  }

**Comment by cryptoni9n** [comment]
ok, can you try using the --dry-run flag with your original send command in combination with lower --fee-rates until you find a value that is accepted?  The --dry-run flag will stop the tx from being signed or broadcasted, so it acts as a test to find a fee-rate that the wallet will allow.  

Examples:
- `ord --cookie-file "D:\bitcoin-26.1\Bitcoin.cookie" wallet send --dry-run --fee-rate 40 3BDRJ8EMsdCZJ6PFJQCSdhBkpx4xn8Ho9H 2000000sats`
- `ord --cookie-file "D:\bitcoin-26.1\Bitcoin.cookie" wallet send --dry-run --fee-rate 35 3BDRJ8EMsdCZJ6PFJQCSdhBkpx4xn8Ho9H 2000000sats`
- `ord --cookie-file "D:\bitcoin-26.1\Bitcoin.cookie" wallet send --dry-run --fee-rate 30 3BDRJ8EMsdCZJ6PFJQCSdhBkpx4xn8Ho9H 2000000sats`
- `ord --cookie-file "D:\bitcoin-26.1\Bitcoin.cookie" wallet send --dry-run --fee-rate 25 3BDRJ8EMsdCZJ6PFJQCSdhBkpx4xn8Ho9H 2000000sats`

test to see at what --fee-rate value you no longer get this insufficient funds message.  this should give us more information as to why the system things this tx will be so expensive.

**Comment by dkofoed** [comment]
Got it figured out ... thanks for the help!
